espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/http/Router.java

```diff
@@ -178,6 +179,8 @@ public BaseResponse route(String uri, Method method, Map<String, String> params,
             return new ErrorResponse(NanoHTTPD.Response.Status.NOT_FOUND, e.getMessage());
         } catch (NotYetImplementedException e) {
             return new ErrorResponse(NanoHTTPD.Response.Status.NOT_IMPLEMENTED, e.getMessage());
+        } catch (StaleElementException e) {
+            return new AppiumResponse<>(AppiumStatus.STALE_ELEMENT_REFERENCE, e.getMessage());
```

Comment:
probably, we could add  e.printStackTrace() instead to simplify problems investigation in logcat output